src/components/dialogs/parameters/network-parameters.js

```diff
@@ -29,13 +29,12 @@ export const FluxConventions = {
 
 export const NetworkParameters = ({ hideParameters }) => {
     const classes = useStyles();
-    const [fluxConventionLocal, handleChangeFluxConvention] = useParameterState(
+
+    const [, handleChangeFluxConvention] = useParameterState(
```

Comment:
remove ,

Reply:
I can't as I want to get the second object in the array returned by the function, not the first